e2e/e2e_test.go

```diff
@@ -176,6 +176,22 @@ var applicationStatusTests = []struct {
 		Result: "SUCCESS",
 		Logs:   applicationStatusLogs,
 	}},
+	{"expected/applicationstatus/simple-success-no-logs.yml", v1.ApplicationStatus{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "simple-success-no-logs-3",
+			Namespace: "default",
+			Labels: map[string]string{
+				"app":                 "simple-success-no-logs",
+				"fiaas/deployment_id": "3",
+			},
+		},
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "fiaas.schibsted.io/v1",
+			Kind:       "ApplicationStatus",
+		},
+		Result: "SUCCESS",
+		Logs: "",
```

Comment:
@oyvindio Is this (and the file below) what you meant by "includeing a test where ApplicationStatus doesn’t have logs set"?

Reply:
Yes, I was wondering if a ApplicationStatus resource like that and/or not having the `log` field present at all would deserialize successfully using this client.

As a sidenote I think there is a minor error in this test case since Logs should be a string slice, not string.